tests/utils_test/uncertainty_metrics_test.py

```diff
@@ -0,0 +1,241 @@
+        repertoire=repertoire,
+        empty_corrected_repertoire=empty_corrected_repertoire,
+        scoring_fn=scoring_fn,
+        num_reevals=num_reevals,
+        random_key=random_key,
+    )
+    pytest.assume(
+        jnp.allclose(
+            corrected_repertoire.fitnesses, repertoire.fitnesses, rtol=1e-05, atol=1e-05
+        )
+    )
+    zero_fitnesses = jnp.where(
+        repertoire.fitnesses > -jnp.inf,
+        0.0,
+        -jnp.inf,
+    )
+    pytest.assume(
+        jnp.allclose(
+            fit_reproducibility_repertoire.fitnesses,
+            zero_fitnesses,
+            rtol=1e-05,
+            atol=1e-05,
+        )
+    )
+    pytest.assume(
+        jnp.allclose(
+            desc_reproducibility_repertoire.fitnesses,
+            zero_fitnesses,
+            rtol=1e-05,
+            atol=1e-05,
+        )
+    )
+
+    # Second, init a Brax environment
+    env_name = "walker2d_uni"
+    episode_length = 100
+    policy_hidden_layer_sizes = (64, 64)
+    env = environments.create(env_name, episode_length=episode_length)
+
+    # Init policy network
+    policy_layer_sizes = policy_hidden_layer_sizes + (env.action_size,)
+    policy_network = MLP(
+        layer_sizes=policy_layer_sizes,
+        kernel_init=jax.nn.initializers.lecun_uniform(),
+        final_activation=jnp.tanh,
+    )
+
+    # Init population of controllers
+    random_key, subkey = jax.random.split(random_key)
+    keys = jnp.repeat(jnp.expand_dims(subkey, axis=0), repeats=batch_size, axis=0)
+    fake_batch = jnp.zeros(shape=(batch_size, env.observation_size))
+    init_policies = jax.vmap(policy_network.init)(keys, fake_batch)
+
+    # Define the fonction to play a step with the policy in the environment
+    def play_step_fn(
+        env_state: EnvState,
+        policy_params: Params,
+        random_key: RNGKey,
+    ) -> Tuple[EnvState, Params, RNGKey, QDTransition]:
+
+        actions = policy_network.apply(policy_params, env_state.obs)
+
+        state_desc = env_state.info["state_descriptor"]
+        next_state = env.step(env_state, actions)
+
+        transition = QDTransition(
+            obs=env_state.obs,
+            next_obs=next_state.obs,
+            rewards=next_state.reward,
+            dones=next_state.done,
+            actions=actions,
+            truncations=next_state.info["truncation"],
+            state_desc=state_desc,
+            next_state_desc=next_state.info["state_descriptor"],
+        )
+
+        return next_state, policy_params, random_key, transition
+
+    # Create the initial environment states for samples and final indivs
+    reset_fn = jax.jit(jax.vmap(env.reset))
+    random_key, subkey = jax.random.split(random_key)
+    keys = jnp.repeat(jnp.expand_dims(subkey, axis=0), repeats=batch_size, axis=0)
+    init_states = reset_fn(keys)
+
+    # Create the scoring function
+    bd_extraction_fn = environments.behavior_descriptor_extractor[env_name]
+    scoring_fn = functools.partial(
```

Comment:
you can do `scoring_fn: Callable = functools.partial(` to avoid the styling error.